packages/react-icons/scripts/build.js

```diff
@@ -45,7 +45,11 @@ async function convertIconData(svg, multiColor) {
         const newName = camelcase(name);
         switch (newName) {
           case "fill":
-            if (attribs[name] === "none" || multiColor) {
+            if (
+              attribs[name] === "none" ||
+              attribs[name] === "currentColor" ||
+              multiColor
+            ) {
```

Comment:
`fill` attributes are defined as `currentColor` in `path` tags in css.gg icons.


```xml
<svg
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  xmlns="http://www.w3.org/2000/svg"
>
  <path d="M5 5H19V19H16L16 8H5V5Z" fill="currentColor" />
  <path
    fill-rule="evenodd"
    clip-rule="evenodd"
    d="M10 19C12.7614 19 15 16.7614 15 14C15 11.2386 12.7614 9 10 9C7.23858 9 5 11.2386 5 14C5 16.7614 7.23858 19 10 19ZM10 16C11.1046 16 12 15.1046 12 14C12 12.8954 11.1046 12 10 12C8.89543 12 8 12.8954 8 14C8 15.1046 8.89543 16 10 16Z"
    fill="currentColor"
  />
</svg>
```

The icons are not displayed because this attribute is not available after the build.

```js
module.exports.CgAbstract = function (props) {
  return GenIcon(
    {
      "tag":"svg","attr":{"viewBox":"0 0 24 24","fill":"none"},

      "child":[
        {
          "tag":"path",
          "attr":{"d":"M5 5H19V19H16L16 8H5V5Z"}
        },
        {
          "tag":"path",
          "attr":{
            "fillRule":"evenodd",
            "clipRule":"evenodd",
            "d":"M10 19C12.7614 19 15 16.7614 15 14C15 11.2386 12.7614 9 10 9C7.23858 9 5 11.2386 5 14C5 16.7614 7.23858 19 10 19ZM10 16C11.1046 16 12 15.1046 12 14C12 12.8954 11.1046 12 10 12C8.89543 12 8 12.8954 8 14C8 15.1046 8.89543 16 10 16Z"
          }
        }
      ]
    }

  )(props);
};
```

![image](https://user-images.githubusercontent.com/1294640/85386133-e1947900-b54b-11ea-8f9f-cafb513fe3b9.png)


https://github.com/react-icons/react-icons/blob/master/packages/react-icons/scripts/build.js#L47-L50

```js
case "fill":
  if (attribs[name] === "none" || multiColor) {
    obj[newName] = attribs[name];
  }
```
If `attribs[name] === "currentColor" ` is added to the above section, then the problem is solved.

![image](https://user-images.githubusercontent.com/1294640/85387186-308ede00-b54d-11ea-8da4-1539028c54b8.png)